__tests__/src/components/WindowTopBar.test.js

```diff
@@ -20,22 +21,10 @@ describe('WindowTopBar', () => {
         removeWindow={mockRemoveWindow}
         classes={{}}
       />,
-    );
+    ).dive();
   });
 
   it('renders without an error', () => {
-    expect(topBar.find('WithStyles(Toolbar)')
-      .dive()
-      .find('WithStyles(Typography)')
-      .dive()
-      .dive()
-      .text()).toBe('Fixture Label');
-    expect(topBar.find('WithStyles(Button).mirador-window-close'));
-  });
-
-  it('calls the removeWindow prop when the close button is clicked', () => {
-    topBar.find('WithStyles(Button)').simulate('click');
-    expect(mockRemoveWindow).toHaveBeenCalledTimes(1);
```

Comment:
Do we need to maintain this test expectation somewhere?

Reply:
I assume that it is already tested in the material ui library. The <WindowTopBar> test should only make sure that the right function is provided to the <Button>. I set out to refactor the component tests after I finished the component-container-stuff.

Reply:
updated the test